ert_storage/database_schema.py

```diff
@@ -1,13 +1,16 @@
 from enum import Enum
-from typing import Any, Mapping, Optional
+from typing import Any, Optional, Mapping
 
 import sqlalchemy as sa
+import uuid
+
 from sqlalchemy.orm import relationship
 from sqlalchemy.sql import func
 
 from ert_storage.database import Base
 
 from ert_storage.ext.sqlalchemy_arrays import StringArray, FloatArray
+from ert_storage.ext.guid import GUID
```

Comment:
GUID is a Microsoft non-standard thing. UUID generally refers to the IETF 4122, which is what Python, SQLAlchemy and PostgreSQL all use. We should not use the name `GUID` since these aren't GUIDs.

https://stackoverflow.com/a/6953207

Reply:
Any suggestion for a name to use?

Reply:
Made a suggestion, please have a look